recipe/meta.yaml

```diff
@@ -32,6 +33,7 @@ outputs:
         - python -c "import cmiles; print(cmiles.__version__)"
 
   - name: cmiles
```

Comment:
I think `cmiles` is way simpler. See [matplotlib](https://github.com/conda-forge/matplotlib-feedstock/blob/master/recipe/meta.yaml#L72-L83).

Reply:
I managed to confuse myself with all the moving parts - in #2 I fixed the version patch, which was the only reason the `build.sh` script needed to exist (although that shouldn't behave much different with the pip one-liner vs the script). But yeah, some of these lines could probably be removed, at last the version check

Reply:
Ah, I think I understand things a little better now. I stared at MPL's recipe for a long time this morning, but I only recently put together that only the base package is actually built, and the second is just adding in the other dependency.